registry/github_registry.go

```diff
@@ -38,6 +39,7 @@ type githubRegistry struct {
 	format         common.RegistryFormat
 	credentialName string
 	service        GithubRepositoryService
+	httpClient     *http.Client
```

Comment:
Well the concern is that it does not simply return the response since it
does some json encoding and things like .prov file are not json so I'd
expect that to not really work.

On Thu, Feb 4, 2016 at 11:10 AM, Jack Greenfield <notifications@github.com>
wrote:

> In registry/github_registry.go
> <https://github.com/kubernetes/deployment-manager/pull/214#discussion_r51921614>
> :
>
> > @@ -38,6 +39,7 @@ type githubRegistry struct {
> >  	format         common.RegistryFormat
> >  	credentialName string
> >  	service        GithubRepositoryService
> > +	httpClient     *http.Client
>
> You're right. It's package private. However, when looking for a way to get
> at the client, I noted this method
> <https://github.com/google/go-github/blob/master/github/github.go#L313>.
> Do we need the client for anything other than Do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/deployment-manager/pull/214/files#r51921614>
> .
>